ui/react-app/src/views/ViewStatus.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Chip,
+  Container,
+  List,
+  ListItem,
+  SxProps,
+  Table,
+  TableCell,
+  tableCellClasses,
+  TableContainer,
+  TableRow,
+  TextareaAutosize,
+  Theme,
+  Typography,
+} from '@mui/material';
+import { useAMStatus } from '../client/am-client';
+
+const tableStyle: SxProps<Theme> = {
+  [`& .${tableCellClasses.root}`]: {
+    borderBottom: 'none',
+  },
+};
+
+const tableHeaderStyle: SxProps<Theme> = {
+  fontWeight: 'bold',
+};
+
+interface tableCellProperties {
+  header: string;
+  content: string;
+}
+
+function CustomTableCell(props: tableCellProperties) {
+  const { header, content } = props;
+  return (
+    <TableRow>
+      <TableCell variant="head" sx={tableHeaderStyle}>
+        {header}
+      </TableCell>
+      <TableCell>{content}</TableCell>
+    </TableRow>
+  );
+}
+
+export default function ViewStatus() {
+  const { data } = useAMStatus();
+  if (data === undefined || data === null) {
+    return <></>;
```

Comment:
Any particular reason why we're returning an empty fragment here iso `null`?

Reply:
not particularly reason :). I will change it by `null`